scripts/us_census/pep/pep_by_srh/README.md

```diff
@@ -5,9 +5,9 @@
 This import covers US Census Population estimate by SRH (Sex, Race and Hispanic / Origin).
 Data is covered for 
 
-    1990 - 2020     County
-    1980 - 2020     State
-    1980 - 2020     National
+    1990 - latest_data     County
```

Comment:
Explain what is referred to as "latest_data" here. You can mention the data published from the source URL. And provide the source URL here.

Reply:
Addressed the latest_data, and added the url.